### PR TITLE
chore(layers-panel): added unit tests

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

File: src/components/comparison/layers-panel/building-explorer.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import { act, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
```
Comment (belom88) [inline src/components/comparison/layers-panel/building-explorer.spec.tsx:2]: Warning

File: src/components/comparison/layers-panel/layers-control-panel.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
+import { act, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
```
Comment (belom88) [inline src/components/comparison/layers-panel/layers-control-panel.spec.tsx:2]: Warning

**Review by belom88 — APPROVED** [review]

File: src/components/comparison/layers-panel/layers-control-panel.spec.tsx (hunk 2 above)
Comment (dryabinin-actionengine) [reply src/components/comparison/layers-panel/layers-control-panel.spec.tsx]: done

File: src/components/comparison/layers-panel/building-explorer.spec.tsx (hunk 1 above)
Comment (dryabinin-actionengine) [reply src/components/comparison/layers-panel/building-explorer.spec.tsx]: done

**Pull request merged**